components/skus/browser/rs/lib/src/sdk/credentials/fetch.rs

```diff
@@ -373,12 +384,13 @@ where
                                 // for batch verification
                                 if bc.encode_base64() == sbc.encode_base64() {
                                     bucket_blinded_creds.push(*bc);
-                                    for t in &item_creds.creds{
+                                    for t in &item_creds.creds {
                                         // find the original token from our list of creds
                                         // that matches up with this blinded token so we can
                                         // add to our bucket creds for verification
-                                        if t.blind().encode_base64() == bc.encode_base64(){
-                                            bucket_creds.push(Token::from_bytes(&t.to_bytes()).unwrap());
+                                        if t.blind().encode_base64() == bc.encode_base64() {
+                                            bucket_creds
+                                                .push(Token::from_bytes(&t.to_bytes()).unwrap());
```

Comment:
The unwrap is ok here. The `Token` from the iterator is being serialized and then deserialized, maybe because it doesn't implement `clone`. Conversion is unlikely to fail therefore. Currently that's just length checks, so the compiler might see though the whole thing.

It's only being flagged here because of the code reformatting. However it is bad style and tedious to have to re-analyse it every time it comes up. Ideally we'd add an `InternalError::TokenError` variant to wrap the underlying type from challenge-bypass-ristretto, but you can replace the `.unwrap()` with
```rust
.or(Err(InternalError::SerializationFailed))?
```
as a quick fix to silence the lint.